specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```diff
@@ -5045,6 +5229,69 @@
       },
       "description": "Filters limit replication to a subset of blobs within the storage account. A logical OR is performed on values in the filter. If multiple filters are defined, a logical AND is performed on all filters."
     },
+    "ObjectReplicationRestorePoints": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ObjectReplicationRestorePoint"
+          },
+          "description": "The replication restore points of storage account."
+        }
+      },
+      "description": "List of storage account object replication restore points."
+    },
+    "ObjectReplicationRestorePointMarker": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Restore point id is auto-generated for each new restore point marker of account."
```

Comment:
should the "id" be readonly?

Reply:
reverted this change from PR it is not in scope for this release.